src/common/logging.cpp

```diff
@@ -77,8 +78,14 @@ void otbrLogSetLevel(otbrLogLevel aLevel)
     sLevel = aLevel;
 }
 
+/** Enable/disable logging with syslog */
+void otbrLogSyslogSetEnabled(bool aDisabled)
```

Comment:
```suggestion
void otbrLogSyslogSetEnabled(bool aEnabled)
```

Better to flip the meaning of this parameter to avoid confusion.